cluster/src/test/scala/org/apache/spark/sql/execution/SnappyTableMutableAPISuite.scala

```diff
@@ -999,4 +999,68 @@ class SnappyTableMutableAPISuite extends SnappyFunSuite with Logging with Before
     }
 
   }
+
+  private def bug2369Test(): Unit = {
+    var snc = new SnappySession(sc)
+    snc.sql("CREATE TABLE SNAPPY_COL_TABLE3(r1 Integer, r2 Integer) " +
+        "USING COLUMN OPTIONS(PARTITION_BY 'R1');")
+    snc.sql("CREATE TABLE SNAPPY_COL_TABLE4(r1 Integer, r2 Integer) " +
+        "USING COLUMN OPTIONS(PARTITION_BY 'R1');")
+
+    snc.insert("SNAPPY_COL_TABLE3", Row(1, 1))
+    snc.insert("SNAPPY_COL_TABLE3", Row(2, 2))
+    snc.insert("SNAPPY_COL_TABLE3", Row(3, 3))
+    snc.insert("SNAPPY_COL_TABLE3", Row(5, 5))
+
+    snc.insert("SNAPPY_COL_TABLE4", Row(1, 1))
+    snc.insert("SNAPPY_COL_TABLE4", Row(2, 2))
+    snc.insert("SNAPPY_COL_TABLE4", Row(3, 3))
+    snc.insert("SNAPPY_COL_TABLE4", Row(4, 4))
+    snc.insert("SNAPPY_COL_TABLE4", Row(5, 5))
+
+
+    SnappyContext.globalSparkContext.stop()
+
+    snc = new SnappySession(sc)
```

Comment:
Why does this test need to stop and restart SparkContext?

Reply:
It was a blind copy-paste from another test in the same class (that may have a legitimate reason to do so.) Will remove it.